Source/WebCore/Configurations/WebCore.xcconfig

```diff
@@ -25,7 +25,15 @@
 #include "../../../Configurations/Version.xcconfig"
 
 GCC_PREFIX_HEADER = WebCorePrefix.h;
-GCC_PREPROCESSOR_DEFINITIONS = $(DEBUG_DEFINES) BUILDING_WEBKIT GL_SILENCE_DEPRECATION=1 GLES_SILENCE_DEPRECATION=1 $(ENABLE_PDFJS) $(ENABLE_LLVM_PROFILE_GENERATION) $(inherited);
+
+WK_USE_JPEGXL = $(WK_USE_JPEGXL_$(CONFIGURATION)_$(WK_PLATFORM_NAME));
+WK_USE_JPEGXL_Production_macosx = $(WK_USE_JPEGXL_Production$(WK_MACOS_1400));
+WK_USE_JPEGXL_Production_MACOS_BEFORE_1400 = YES;
+
+WK_USE_JPEGXL_PREPROCESSOR_DEFINITIONS = $(WK_USE_JPEGXL_PREPROCESSOR_DEFINITIONS_$(WK_USE_JPEGXL));
+WK_USE_JPEGXL_PREPROCESSOR_DEFINITIONS_YES = USE_JPEGXL=1;
+
+GCC_PREPROCESSOR_DEFINITIONS = $(DEBUG_DEFINES) BUILDING_WEBKIT GL_SILENCE_DEPRECATION=1 GLES_SILENCE_DEPRECATION=1 $(ENABLE_PDFJS) $(ENABLE_LLVM_PROFILE_GENERATION) $(WK_USE_JPEGXL_CONFIGURATION) $(WK_USE_JPEGXL_PREPROCESSOR_DEFINITIONS) $(inherited);
```

Comment:
What is $(WK_USE_JPEGXL_CONFIGURATION)? Looks like it might be left behind and can be safely deleted.

Reply:
Huh. I don't know how that got there. Fixed.